### DISCUSSION TITLE
Estágio obrigatório da faculdade vs experiência comprovada na área

### DISCUSSION THREAD

**Discussion by igorthurow** [frontendbr/forum#2042] [Perguntas e Respostas]
Fala galera, beleza?

Me chamo Igor e atuo na área há quase 4 anos, aprendi com cursos livres e estudando de forma não academica. Por motivos pessoais eu gostaria de fazer um **bacharel EAD** em computação, de preferencia ciencias da computação, mas eu só encontro curso com estágio obrigatório para conclusão. 

Hoje eu já não poderia dar essa passo atrás na carreira e fazer um estagio, imagina daqui a 4 anos onde minha senioridade provavelmente será maior que agora 😞 

Tecnólogo não é uma opção, eu quero um curso menos prático e muito teórico, e também tenho um sonho de trabalhar fora do país, então estou optando por bacharel mesmo.

Alguma dica do que pode ser feito? Faculdade que possa flexibilizar nesse ponto?

**Comment by wmartins** [comment]
Opa! Eu não sei em quais faculdades tu olhou, mas eu me formei na PUCRS e me lembro que uma época eles flexibilizavam essa questão.

Andei dando uma olhadas nos materiais e nesse PDF (https://www.pucrs.br/politecnica/wp-content/uploads/sites/166/2019/12/Regulamento-Ativ-Complem-ESCOLA-POLIT%C3%89CNICA_29.11.19.pdf), a partir da página 12, eles detalham que os alunos precisam fazer uma carga de atividades complementares e lá consta "Estágio ou atividades profissionais" como uma das opções, então eles acabam sendo flexíveis nesse ponto.

Trouxe só como um exemplo mesmo, acho que faz um pouco de sentido as faculdades flexibilizarem isso. De qualquer forma, recomendo entrar em contato com cada uma dessas para confirmar.

**Comment by igorthurow** [comment]
Fala @wmartins beleza? Eu havia olhado na Ritter, na Uninter, Estácio e na FIAP. Bom, só de saber que tem meios de flexibilizar ja me tranquiliza, porque eu acabei de ler na ead.com.br que para todas IES seria obrigatório o estágio no curso de ciencias. 
Vou dar mais uma procurada, que acho que na PUC não tem graduação EAD

**Reply by wmartins** [reply]
Realmente, acho que não tem EAD na PUCRS mesmo. Chegou a dar uma olhada na Unisinos também? Sei que eles têm um curso de ciência da computação EAD.

Mas de qualquer forma, eu imagino que as faculdades devem (ao menos na minha cabeça) ser flexíveis com isso.

**Comment by ninetails** [comment]
Se tiver algum curso que te agrade, tente ir atrás da coordenadoria pra ver esta questão. Talvez algumas aceitem uma outra forma como acompanhamento de atividades e relatórios dos teus gestores direto.

(Não generalizando mas o que a) Faculdade quer é que mais pessoas se formem nela pois além de estar vendendo diplomas também gera estatísticas favoráveis para o marketing do curso.
